smartclide-monitoring/src/main/java/eu/smartclide/contexthandling/services/GitlabApiClient.java

```diff
@@ -118,26 +118,32 @@ private int calculateTimeSinceLastCommit(String projectId, JsonObject commit) {
         int difference = 0;
         // check if parent id exists for given commit
         if (commit.get("parent_ids").getAsJsonArray().size() > 0) {
-            String parentCommitId = commit.get("parent_ids").getAsString();
-            String commitCreationDateStr = commit.get("created_at").getAsString();
-            JsonObject parentCommit = getCommitById(projectId, parentCommitId);
-            if (parentCommit.getAsJsonObject().has("created_at")) {
-                String parentCommitCreationDateStr = parentCommit.getAsJsonObject().get("created_at").getAsString();
-
-                try {
-                    ZonedDateTime commitCreationDate = ZonedDateTime.parse(commitCreationDateStr, formatter);
-                    ZonedDateTime parentCommitCreationDate = ZonedDateTime.parse(parentCommitCreationDateStr, formatter);
-                    long longDifference = commitCreationDate.toInstant().getEpochSecond() -
+            String parentCommitId;
+            try {
+                parentCommitId = commit.get("parent_ids").getAsString();
+            } catch (IllegalStateException e) {
+                logger.error("array found, taking first entry: {}", commit.get("parent_ids"));
+                parentCommitId = commit.get("parent_ids").getAsJsonArray().get(0).getAsString();
```

Comment:
`parent_ids` is always an array even if it only contains one value.

So I think we can always use `getAsJsonArray` and avoid the `try-catch`

Reply:
ok, [x] done